
int16 camera_index
int16 point_index
float64 x
float64 y
